addons/website/models/assets.py

```diff
@@ -47,6 +47,7 @@ def make_scss_customization(self, url, values):
                 'header-sales_one-gradient': 'null',
                 'header-sales_two-gradient': 'null',
                 'header-sales_three-gradient': 'null',
+                'header-sales_four-gradient': 'null',
```

Comment:
I guess it's ok but I am wondering why you needed multiple variables to handle a gradient of different templates. Could not you have shared them?

Reply:
Indeed, it's better to use a shared gradient 👍 . I introduced `menu-secondary-gradient` in the `sales_one` commit and reused it in the three other `sales_*` templates.